models/tsodyks_connection_hom.h

```diff
@@ -0,0 +1,331 @@
+   a presynaptic increase of release probability, which is modeled by variable U in Eq (4).
+   The original interpretation of variable y is the amount of glutamate concentration in
+   the synaptic cleft. In [1] this variable is taken to be directly proportional to the
+   synaptic current caused in the postsynaptic neuron (with the synaptic weight w as a
+   proportionality constant). In order to reproduce the results of [1] and to use this
+   model of synaptic plasticity in its original sense, the user therefore has to ensure
+   the following conditions:
+
+   1.) The postsynaptic neuron must be of type iaf_psc_exp or iaf_tum_2000, because
+   these neuron models have a postsynaptic current which decays exponentially.
+
+   2.) The time constant of each tsodyks_synapse targeting a particular neuron
+   must be chosen equal to that neuron's synaptic time constant. In particular that means
+   that all synapses targeting a particular neuron have the same parameter tau_psc.
+
+   However, there are no technical restrictions using this model of synaptic plasticity
+   also in conjunction with neuron models that have a different dynamics for their synaptic
+   current or conductance. The effective synaptic weight, which will be transmitted
+   to the postsynaptic neuron upon occurrence of a spike at time t is u(t)*x(t)*w, where
+   u(t) and x(t) are defined in Eq (3) and (4), w is the synaptic weight specified upon
+   connection.
+   The interpretation is as follows: The quantity u(t)*x(t) is the release probability
+   times the amount of releasable synaptic vesicles at time t of the presynaptic neuron's
+   spike, so this equals the amount of transmitter expelled into the synaptic cleft.
+   The amount of transmitter than relaxes back to 0 with time constant tau_psc of the
+   synapse's variable y.
+   Since the dynamics of y(t) is linear, the postsynaptic neuron can reconstruct from the
+   amplitude of the synaptic impulse u(t)*x(t)*w the full shape of y(t).
+   The postsynaptic neuron, however, might choose to have a synaptic current that is not
+   necessarily identical to the concentration of transmitter y(t) in the synaptic cleft.
+   It may realize an arbitrary postsynaptic effect depending on y(t).
+
+   Parameters:
+     U         double - maximum probability of release [0,1]
+     tau_psc   double - time constant of synaptic current in ms
+     tau_fac   double - time constant for facilitation in ms
+     tau_rec   double - time constant for depression in ms
+     x         double - initial fraction of synaptic vesicles in the readily releasable pool [0,1]
+     y         double - initial fraction of synaptic vesicles in the synaptic cleft [0,1]
+
+  References:
+   [1] Tsodyks, Uziel, Markram (2000) Synchrony Generation in Recurrent Networks
+       with Frequency-Dependent Synapses. Journal of Neuroscience, vol 20 RC50
+
+  Transmits: SpikeEvent
+
+  FirstVersion: March 2006
+  Author: Moritz Helias, Susanne Kunkel
```

Comment:
I did not find any further issues than the ones pointed out by HEP apart from the authorship, as stated above.